hbase_storage/pom.xml

```diff
@@ -136,6 +136,11 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>com.cedarsoftware</groupId>
+          <artifactId>java-util</artifactId>
+          <version>1.61.0</version>
+        </dependency>
```

Comment:
I'd rather not add this dependency, since we are only using .deepEquals() it should be easy to remove.

Reply:
@dgomezferro OK, is it because this is not a well-known library that it's not a good idea to include?  Just trying to know the reason in case I want to include other libraries in the future.  They use the Apache license, so I'm assuming it's not a license issue.

Also, if we don't use YARN in the cloud, can you point me to the code that determines the memory available to start spark executors?  I only have one YARN config setting I'm using that's related to how much memory we can use, which determines the number of spark executors we use.

Reply:
@dgomezferro OK, I've removed the usage and reference to DeepEquals.